gravitee-apim-portal-webui-next/src/entities/categories/categories.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { CategoryLinks } from './category-links';
+
+export interface Categories {
```

Comment:
It would be good to add `?` to each of these properties in case the backend does not return the attributes. Allows us to avoid problems in the front...